### PR TITLE
Master

### PR DESCRIPTION
**Pull request by akshitjain**
wordpress added

### CODE REVIEW

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/blob/master/contributing.md

**Comment by sindresorhus** [comment]
You also have a merge conflict.

**Comment by sindresorhus** [comment]
Closing for lack of response.

File: readme.md [hunk 1]
```diff
@@ -176,6 +176,7 @@
 - [Mad Science Modules](https://github.com/feross/mad-science-modules)
 - [Maintainance Modules](https://github.com/maxogden/maintenance-modules) *(for npm packages)*
```
Comment (The-Only79) [inline readme.md]: 1979-7-9

Comment (The-Only79) [inline readme.md]: 799872420

Comment (The-Only79) [inline readme.md]: 1979-7-9

**Pull request closed without merging**